model_inference/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.11
```

Comment:
Try using python slim images. They will most likely fit your task as well as this one, but will weigh much less and be faster to build since it only has basic python customization.
3.11-slim should work for you.